go.mod

```diff
@@ -0,0 +1,11 @@
+module github.com/google/docsy
+
+go 1.12
+
+//replace github.com/google/docsy/module v0.0.0-20220129185551-e4b42eecdf1d => e4b42eecdf1d
+
+require (
+	github.com/FortAwesome/Font-Awesome v0.0.0-20210804190922-7d3d774145ac // indirect
+	github.com/google/docsy/module v0.2.0-pre // indirect
+	github.com/twbs/bootstrap v4.6.1+incompatible // indirect
```

Comment:
What's the `+incompatible` for?

Reply:
This is automatically generated when issueing `hugo mod get github.com/twbs/bootstrap`.
`+incompatible` means the dependency (=bootstrap) is not a Go module (it doesn't have go.mod in its source code). This doesn't prevent us from pulling it in as dependency, fortunately.